donkeycar/parts/imu.py

```diff
@@ -12,13 +15,36 @@ class Mpu6050:
     sudo python setup.py install
 
     pip install mpu6050-raspberrypi
+    
+    - MPU9250
+    pip install mpu9250-jmdev
+    
     '''
 
-    def __init__(self, addr=0x68, poll_delay=0.0166):
-        from mpu6050 import mpu6050
-        self.sensor = mpu6050(addr)
+    def __init__(self, addr=0x68, poll_delay=0.0166, sensor=SENSOR_MPU6050):
+        self.sensortype = sensor
+        if self.sensortype == SENSOR_MPU6050:
+            from mpu6050 import mpu6050
+            self.sensor = mpu6050(addr)
```

Comment:
If `mpu6050` is a class just change it to `MPU6050`.

That way it's clearer that this is not a invocation, but an instantiation.